galaxy_importer/loaders/collection.py

```diff
@@ -19,7 +19,10 @@
 import os
 import re
 
-import ansible_builder.introspect
+try:
+    import ansible_builder.introspect
+except ImportError:
+    import ansible_builder._target_scripts.introspect
```

Comment:
think would need to be refactored here and below to be `from ansible_builder._target_scripts import introspect` since below it uses

```
python_deps, system_deps = ansible_builder.introspect.process_collection(self.path)
```